packages/react-server/core/test/specRuntime/TransitionPage.js

```diff
@@ -1,5 +1,5 @@
 var React = require("react"),  //eslint-disable-line no-unused-vars
-	Link = require("../../components/Link");
+	Link = require("../../components/Link"); //eslint-disable-line no-unused-vars
```

Comment:
Turns out we were missing some rules we should've enabled: https://github.com/yannickcr/eslint-plugin-react#user-content-recommended-configuration